docs/guides/choosing-a-subject-type.md

```diff
@@ -0,0 +1,89 @@
+# Choosing a Subject Type
+
+Within a Permissions System, a [CheckPermission] call is always made between an object representing the *resource* and an object representing a *subject*: The API call returns whether the *subject* has the specified permission on the *resource*.
+
+[CheckPermission]: https://buf.build/authzed/api/docs/main/authzed.api.v1#CheckPermission
+
+The choice of how to represent subjects can have large downstream effects on the design of a permissions system.
```

Comment:
We probably want to link to the blog post on this somewhere in this section

https://authzed.com/blog/why-model-users/